.vagrant/machines/default/virtualbox/vagrant_cwd

```diff
@@ -0,0 +1 @@
+/home/jdbohrman/Documents/pyswarms
```

Comment:
I'm wondering if the hardcoded paths are necessary.

Reply:
Hey @ljvmiranda921 great to be here! Here's the info I found on it. The documentation is slim but it appears that is the default insecure private key that Vagrant ships with. Here's a couple links: 

https://www.vagrantup.com/docs/vagrantfile/ssh_settings.html
https://github.com/hashicorp/vagrant/issues/8058

Also, I'm fairly sure you're absolutely correct, as running `vagrant up` would provision the config files and others.